Doc/whatsnew/3.13.rst

```diff
@@ -393,6 +393,10 @@ Deprecated
 
   (Contributed by Victor Stinner in :gh:`105145`.)
 
+* Deprecate the :c:func:`PyImport_ImportModuleNoBlock` function which is just
+  an alias to :c:func:`PyImport_ImportModule` since Python 3.3.
+  (Contributed by Victor Stinner in :gh:`105396`.)
```

Comment:
Shall we include the scheduled removal version here? It will help people plan updating their code.

Reply:
Sure, done. It was already done in the Doc/c-api/ change.